doc/api/Player_Events.md

```diff
@@ -536,6 +536,25 @@ video track when in directfile mode to avoid that case (this is documented
 in the corresponding APIs).
 </div>
 
+### noPlayableTrack
+
+_payload type_: `Object`
+
+Emitted when no tracks of a particular type can be selected for a period.
```

Comment:
We might want to indicate that this happen when the corresponding options are set, and in which condition it arises

Else a developer won't understand what this event is for and whether it should handle it